test/example_tests/objective_c_example_test.dart

```diff
@@ -22,9 +19,21 @@ void main() {
     });
 
     test('objective_c', () {
-      final pubspecFile = File('example/objective_c/pubspec.yaml');
-      final pubspecYaml = loadYaml(pubspecFile.readAsStringSync()) as YamlMap;
-      final config = Config.fromYaml(pubspecYaml['ffigen'] as YamlMap);
```

Comment:
This means we have the config in two places now, how are we going to keep that sync? These tests are here to ensure our examples keep working.

What is the reason for moving the config here instead of reading it from the file?

Reply:
Yes, I brought this up with Liam as a potential point of contention offline and we discussed a solution. My proposed solution is to make an additional test helper for reading a config from a file given its path from the project root and to use separate config files instead of a key in pubspec.yaml, so that the config file helper can simply read the file into a string and then use the existing helper, rather than parsing the pubspec.yaml to get the ffigen key

Reply:
sgtm